applications/osb-portal/src/pages/WorkspacePage.tsx

```diff
@@ -149,10 +149,23 @@ export const WorkspacePage = (props: any) => {
   const [editWorkspaceOpen, setEditWorkspaceOpen] = React.useState(false);
   const [refresh, setRefresh] = React.useState(true);
   const [error, setError] = React.useState<any>(null);
+  const [options, setOptions] = React.useState<string[]>([]);
+
+  const PREFIX_TEXT = "Open with "
 
   React.useEffect(() => {
     WorkspaceService.getWorkspace(parseInt(workspaceId, 10)).then((ws) => {
       setWorkspace(ws);
+
+      // Populate ordered list from OSBApplications
+      // set last used application as first entry if it is set
+      let apps = ws ? ws.lastOpen.type.application.name ? [PREFIX_TEXT + ws.lastOpen.type.application.name] : [] : null;
```

Comment:
Please run `npm run prebuild` to ensure the linting is ok!

Tip: `npm run prebuild-dev` does some fixes for you.

```
$ npm run prebuild

> frontend@0.4.0 prebuild /home/user/OSBv2/applications/osb-portal
> tslint -c tslint.json 'src/**/*.{ts,tsx}'

src/pages/WorkspacePage.tsx:162:11
ERROR: 162:11  prefer-const  Identifier 'apps' is never reassigned; use 'const' instead of 'let'.

npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! frontend@0.4.0 prebuild: `tslint -c tslint.json 'src/**/*.{ts,tsx}'`
npm ERR! Exit status 2
npm ERR! 
npm ERR! Failed at the frontend@0.4.0 prebuild script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/user/.npm/_logs/2022-03-22T11_52_11_405Z-debug.log
```

Reply:
I have linting enabled already, so it gets done as I type. I did see this but we use `apps.push` in the next few lines, so apps is not being assigned to, but it is being modified. Should it still be `const`?

Reply:
Yeh, looks like `const` means const reference, so`array.push` can still be used. I'll go update this:

https://www.w3schools.com/jS/js_array_const.asp